packages/stepper/src/components/Stepper.connect.js

```diff
@@ -0,0 +1,9 @@
+import { connect } from 'react-redux';
+import { default as Stepper } from './Stepper.component';
+import { getStepsForResource } from '../service/Stepper.selectors';
+
+export const mapStateToProps = (state, ownProps) => ({
+	steps: getStepsForResource(state, ownProps.resourceType, ownProps.resourceId),
+});
+
+export default connect(mapStateToProps)(Stepper);
```

Comment:
this is ok to have a container in a components folder? 
i know that will be strange to have a folder containers with only this file but in this case is also strange for me :s

Reply:
I will move into a new folder :+1: